resources/views/components/textarea.blade.php

```diff
@@ -49,6 +49,7 @@
 <div class="relative w-full @if($add_clearing) mb-3 @endif">
     @if($toolbar)
         <div id="{{$name}}">{{$selected_value}}</div>
+        <input type="hidden" name="{{ $name }}" id="{{ $name }}-hidden" value="">
```

Comment:
Please change this to a textarea thats hidden and has class="size-0". Input fields have a limit to how much text they can contain @prince-noman

Reply:
Sure, let me fix it. I tried it with a large text content and found no limitations. That hidden input is only for supplying the name attribute in the request. I appreciate your suggestion. @mkocansey